.github/workflows/ci.yml

```diff
@@ -68,6 +68,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Set up JDK 11
```

Comment:
The new AS needs java 11

Reply:
Can this be put under `defaults` to make it run for each stage instead of duplicating it?

Reply:
TIL! perfect :)

Reply:
actually it doesn't look like you can add a step in `defaults` :(

Reply:
Ah yeah I was juts looking at [the docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions?query=defaults#defaults) 😢 
I guess we'd have to create a separate action, which is probably overkill